content/de/api_gateway.md

```diff
@@ -0,0 +1,27 @@
+---
+title: API Gateway
+status: Feedback erwünscht
+category: Technologie
+---
+
+## Was es ist
+Ein [API](/application_programming_interface/) Gateway ist ein Werkzeug zur Zusammenfassung einzelner
+Applikations APIs an einem einzigen Ort. Es erlaubt Organisationen Kernfunktionen wie 
+Authentifikation, Autorisation oderauch die Limitierung von Anfragen in zentral zu managen.
```

Comment:
in zentral zu managen.
Ich glaube das in muss weg

Reply:
jo